lib/game.rb

```diff
@@ -50,11 +50,16 @@ def selection(game_options_descriptions)
   position_validator = PositionValidator.new(board)
   parse_input = ParseInput.new
   user = User.new(display: display, parse_input: parse_input)
-  human_player = HumanPlayer.new(user: user, token: "X", validator: position_validator)
-  computer_player = ComputerPlayer.new(token: "O", board: board)
+  X = "X"
+  O = "O"
```

Comment:
Where does this constant get defined? Is it defined globally? If this was inside a class, it would be localized.

```ruby
class SomeClass
  X = "X"
end

SomeClass::X
```

I wonder if it needs to be constant here in the `main`. Is there a benefit over using a normal variable?

Reply:
I think normal variable is fine.  No constant needed.